### PR TITLE
Fixes #8 Remote url() assets should not be given to asset_path

### LINKED ISSUE #8: Remote http(s) url() assets not ignored
Hi,

I'm trying to use Torba wth jQuery and it looks like it doesn't work.
Here is my Torbafile:
```
gh_release source: 'jquery/jquery', tag: '2.0.3'
```

When I try to do `torba pack` I get that error:
`Couldn't find an asset with path '/Users/admin/.torba/cache/jquery-bad25f43/test/data/http:/static.jquery.com/files/rocker/images/logo_jquery_215x53.gif'.`

This file seem to be requested here:
```
/Users/admin/.torba/cache/jquery-bad25f43/test/data/testsuite.css
144:body, div { background: url(http://static.jquery.com/files/rocker/images/logo_jquery_215x53.gif) no-repeat -1000px 0; }
```

Am I missing something? My understanding is that this file should not be requested by torba because it's a remote url anyway. And asset_url() is not gonna work on it.

### PR DESCRIPTION
**Pull request by pcboy**
Fixes #8 
It looks like the regexp used was incorrect. I have no more issues with this patch.

### CODE REVIEW

**Comment by nashbridges** [comment]
Could you please add a test case for that?

**Comment by pcboy** [comment]
Sure. I just added one and everything looks fine!

File: test/css_url_to_erb_asset_path_test.rb [hunk 1]
```diff
@@ -14,6 +14,17 @@ def test_no_url
     def test_url_with_data
       css = "background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaskip);"
       assert_equal css, filter.call(css, "/current/file")
+      css = "background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaskip');"
+      assert_equal css, filter.call(css, "/current/file")
```
Comment (nashbridges) [inline test/css_url_to_erb_asset_path_test.rb]: Is this accidental duplication?

Comment (pcboy) [inline test/css_url_to_erb_asset_path_test.rb]: I was just checking that either data( or data(' works. Because before my patch it wasn't.

Comment (nashbridges) [inline test/css_url_to_erb_asset_path_test.rb]: Oh! I didn't notice quotes.
Do you mind to split this into separate commit then? Because it is not related to your "http://" problem.

Comment (pcboy) [inline test/css_url_to_erb_asset_path_test.rb]: No problem. Split it up in two commits. I can make a different PR for the second commit if you want.

Comment (nashbridges) [inline test/css_url_to_erb_asset_path_test.rb]: I prefer to cherrypick commits instead of merge, thus there's no need for a separate PR

**Comment by nashbridges** [comment]
[Released!](https://rubygems.org/gems/torba/versions/0.4.2)
Thanks for your contributions.
